app/dashboard/models.py

```diff
@@ -1065,6 +1078,60 @@ def stats(self):
             (success_rate, 'Success Rate'),
         ]
 
+    @property
+    def get_quarterly_stats(self):
+        user_active_in_last_quarter = False
+        last_quarter = datetime.now() - timedelta(days=90)
+        bounties = self.bounties.filter(modified_on__gte=last_quarter)
+        fulfilled_bounties = [
+            bounty for bounty in bounties if bounty.is_hunter(self.handle) and bounty.status == 'done'
+        ]
+        fulfilled_bounties_count = len(fulfilled_bounties)
+        total_earned_eth = sum([
+            bounty.value_in_eth if bounty.value_in_eth else 0
+            for bounty in fulfilled_bounties
+        ])
+        total_earned_eth /= 10**18
+        total_earned_usd = sum([
+            bounty.value_in_usdt if bounty.value_in_usdt else 0
+            for bounty in fulfilled_bounties
+        ])
+
+        num_completed_bounties = bounties.filter(idx_status__in=['submitted', 'done']).count()
+        total_bounties = bounties.count()
+        completetion_percent = int(
+            round(num_completed_bounties * 1.0 / total_bounties, 2) * 100
+        ) if total_bounties != 0 else '0'
```

Comment:
This needs to be updated to `if total_bounties != 0 else 0`
Else it would throw an error below at 1113